server/pulp/server/db/manage.py

```diff
@@ -274,3 +274,8 @@ def _start_logging():
     console_handler = logging.StreamHandler()
     console_handler.setLevel(logging.INFO)
     _logger.root.addHandler(console_handler)
+    # Django will un-set our default ignoring DeprecationWarning *unless* sys.warnoptions is set.
+    # So, set it as though '-W ignore::DeprecationWarning' was passed on the commandline. Our code
+    # that sets DeprecationWarnings as ignored also checks warnoptions, so this must be added after
+    # pulp.server.logs.start_logging is called but before Django is initialized.
```

Comment:
This comment is written as though #2910 is already merged. The code's functionality doesn't actually depend on the change in that PR for EL7, which is what issue 1594 is concerned with, but referencing the code where we're explicitly ignoring those warnings seemed more useful to developers than any implicit DeprecationWarning visibility settings inherent in python.